i18n/fr/docusaurus-plugin-content-docs/version-23.04/api/clapi.md

```diff
@@ -887,13 +887,12 @@ Arguments are composed of the following columns:
 | 2     | Name of the I/O object      |
 | 3     | Nature of I/O object        |
 
-#### Delinput, Deloutput and Dellogger
+#### Delinput and Deloutput
 
 In order to remove an I/O object from the Centreon Broker configuration, use one of the following commands:
 
 * **DELINPUT**
 * **DELOUTPUT**
-* **DELLOGGER**
 
 Example:
 
```

Comment:
Missing example update:
`centreon -u admin -p 'centreon' -o CENTBROKERCFG -a deloutput -v "broker cfg for poller test;4"`